examples/child-design-system/README.md

```diff
@@ -0,0 +1,103 @@
+## Introduction
+
+There are many independent product teams across different contracts and companies that work under the same CMS sites (i.e. Medicare.gov, Healthcare.gov).
+
+These product teams need a shared resource specific to those sites. It’s not possible or desirable for the CMSDS core to cover those site specific case.
+
+This is the reasoning behind creating the CMS design system family which consists of the CMSDS core and child design systems.
+
+## Getting started
+
+1. Download this child design system example folder
+1. Run `yarn install` to install dependencies
```

Comment:
Number the points

Reply:
Markdown will automatically at the number needed, so you can use `1.` for all the list items. See the rendered version here - https://github.com/CMSgov/design-system/pull/740/files?short_path=8bb03a6#diff-8bb03a618d7ed005710786e52c92b146